docs/source/models/other_models/pregnancy/gbd_2021_mncnh/index.rst

```diff
@@ -0,0 +1,335 @@
+
+GBD Modeling Strategy
+----------------------
+
+Pregnancy and births are not explicit outcomes in the GBD study. However, there are location- and year-specfic GBD covariates related to fertility and births, including:
+
+.. list-table:: Covariates
+  :widths: 15 15 15
+  :header-rows: 1
+
+  * - Covariate name
+    - Covariate ID
+    - Note
+  * - Live births (thousands)
+    - 60
+    - 
+  * - Live births by sex
+    - 1106
+    - 
+  * - Live births by sex and maternal age
+    - 2298
+    - 
+  * - Stillbirth to live birth ratio
+    - 2267
+    - Not specific to maternal age. Upper and lower bound estimates are equal to mean estimate (no uncertainty interval)
+  * - Age-specific fertility rate
+    - 13
+    - 
+  * - Total fertility rate
+    - 2363
+    - 
+
+GBD has estimated the prevalence of pregnancy (as an intermediate variable for the estimation of various outcomes) as:
+
+.. math::
+
+   (ASFR + (SBR * ASFR)) * 46/52
+
+Where :math:`ASFR` is the age-specific fertility rate, :math:`SBR` is the stillbirth to live birth ratio, and :math:`46/52` is the proportion of the year spent pregnant (40 weeks) and postpartum (6 weeks).
+
+.. todo::
+
+   Determine the threshold of gestational age at which a loss of pregnancy is classified as a stillbirth rather than miscarriage for the GBD covariate. Standard thresholds are 20 or 24 weeks.
+
+Vivarium Modeling Strategy
+----------------------------
+
+We will model pregnancy as a characteristic of women of reproductive age in our simulations. We will inform the *incidence* of pregnancy using the age-specific fertility and stillbirth to live birth ratio covariates from GBD. We will inform the *duration* of pregnancy using the GBD 2019 exposure distribution of gestational age.
+
+.. image:: diagram.svg
+
+.. note::
+
+  Maternal disorders are entirely handled in the intrapartum model for wave 1 of this project. At current there are no risk factors (other than age) included either. 
+
+  We will add risk factors to this model (BMI, blood pressure, hemoglobin) and may also add some maternal disorders, such as preeclampsia and anemia. Though we have not determined where we will have simulants accumulate YLDs/deaths (might all be in the intrapartum model). 
+
+.. list-table:: State definitions
+  :widths: 15 15 15
+  :header-rows: 1
+
+  * - State
+    - Description
+    - Note
+  * - np
+    - Not pregnant or postpartum
+    - 
+  * - p
+    - Pregnant
+    - 
+  * - output
+    - Output data 
+    - Recording of simulant data for use in the next phase of the simulation, the intrpartum model.
+
+.. note::
+
+  For this model, we will not be using "time" or "time steps" in the traditional sense. Rather, simulants will be initialized and then go through the decision tree outlined on the :ref:`MNCNH Portfolio project <2024_concept_model_vivarium_mncnh_portfolio>` page. Following that, data will be recorded for use in the intrapartum model. 
```

Comment:
Do we researchers expect that this will use the "individual clock" affordance that the engineers worked on last year?

Reply:
That isn't quite how I was thinking about this. With the individual clocks time still exists in the model, just some simulants can take different size time steps than other simulants. Here, we don't really have "time" in the model, you just jump to "birth" and then we record separately the gestational age and calculate YLDs.